### PR TITLE
setup route for react & angular, created bear-bones ui with routing demo

### PR DESCRIPTION
**Pull request by nateng98**
Setup routing for React and Angular

Update:
- Added React dependencies and fixed indentations

Update:
- Changed Angular webapp from SSR to CSR
  - Remove SSR related files and dependencies

### CODE REVIEW

File: src/guest_webapp/src/pages/AmenitiesPage.jsx
Comment (rainclouded) [inline src/guest_webapp/src/pages/AmenitiesPage.jsx:1]: Please adjust the indentation

File: src/guest_webapp/src/pages/HomePage.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+const HomePage = () => {
+	return (
+		<div className='flex justify-center items-center h-screen flex-col'>
+			<h3>Welcome, Mr. Guest</h3>
+			<div>
+				<Link to='/amenities'>Amenities</Link>
+			</div>
+			<div>
+			<Link to='/'>Log out</Link>
```
Comment (rainclouded) [inline src/guest_webapp/src/pages/HomePage.jsx:12]: Being pedantic, but here too please

**Review by rainclouded — CHANGES_REQUESTED** [review]

**Review by Mondane-Monkeys — APPROVED** [review]
This is amazing, great work!

**Comment by nateng98** [comment]
There is an issue with Angular SSR right now (routing to nginx default on refresh instead of staying on Angular webapp)

**Comment by rainclouded** [comment]
lgtm

**Review by rainclouded — APPROVED** [review]

**Pull request merged**